File: ce/customerengagement/on-premises/snippets/visualbasic/CRMV8/auditingvb/vb/auditing6.vb
' <snippetauditing6>


 ' Get the list of audit partitions.
 Dim partitionRequest As RetrieveAuditPartitionListResponse = CType(_service.Execute(New RetrieveAuditPartitionListRequest()), RetrieveAuditPartitionListResponse)
 Dim partitions As AuditPartitionDetailCollection = partitionRequest.AuditPartitionDetailCollection

' </snippetauditing6>

File: ce/customerengagement/on-premises/snippets/csharp/CRMV8/auditing/cs/auditing6.cs
// <snippetauditing6>


 // Get the list of audit partitions.
 RetrieveAuditPartitionListResponse partitionRequest =
     (RetrieveAuditPartitionListResponse)_service.Execute(new RetrieveAuditPartitionListRequest());
 AuditPartitionDetailCollection partitions = partitionRequest.AuditPartitionDetailCollection;

// </snippetauditing6>